test/FastTests/Sparrow/Hashing.cs

```diff
@@ -315,54 +256,11 @@ public static IEnumerable<object[]> BufferSize
                     new object[] {128},
                     new object[] {129},
                     new object[] {1000},
+                    new object[] {Random.Shared.Next(10000)}
```

Comment:
I think we won't see the number returned here once a test will fail. We have `[InlineDataWithRandomSeed]` for that purpose

Reply:
it should yes, but I like the usage of `[InlineDataWithRandomSeed]` more.